history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -1186,8 +1264,11 @@ so that simplicity of life is what they observe with great satisfaction;
 though it sometimes happens not to be without the
 mockeries of the vain world they live in.
 
+[.signed-section-signature]
```

Comment:
Wasn't sure if I should do something like this, or remove "Thus far" and just leave his signature.

Reply:
Yeah, that's super weird.  🤔  How about we actually bring it up to join with the paragraph above, and italicize it, like so:

```ASCIIDOC
mockeries of the vain world they live in.
__Thus far William Penn.__^
```